packages/utils/src/collections.ts

```diff
@@ -0,0 +1,9 @@
+export function zip<A, B, C>(fn: (a: A, b: B) => C, as: A[], bs: B[]) {
```

Comment:
What do you think about just using `lodash`s zip?

Reply:
lodash is a pretty big library, if we only need just a couple of thing i woudl try to avoid it

Reply:
We already have it in `@celo/utils` :)